### PR TITLE
Add kafka0-staging

### PR DESCRIPTION
**Pull request by dannyroberts**
I'm going to migrate kafka/zookeeper on staging before doing it on production

### CODE REVIEW

File: environments/staging/terraform.yml [hunk 1]
```diff
@@ -91,6 +91,13 @@ servers:
     az: "a"
     volume_size: 80
 
+  - server_name: "kafka0-staging"
+    server_instance_type: t3.medium
+    network_tier: "db-private"
```
Comment (pr33thi) [inline environments/staging/terraform.yml]: Should this be app-private?

Comment (dannyroberts) [reply environments/staging/terraform.yml]: Nope, app-private is for the app layer, things that run the code that we write for commcarehq; kafka is in the db layer, it's a third-party service for general use that we rely on, analogous to the way we use redis, postgres, elasticsearch, couchdb, for different purposes.

Comment (mkangia) [reply environments/staging/terraform.yml]: Is there a documentation to understand what these fields and their values mean?

Comment (pr33thi) [reply environments/staging/terraform.yml]: I have an explanation on this page: https://confluence.dimagi.com/display/internal/How+to+migrate+a+rackspace+machine+to+AWS
Explained in step 1 of 'Create and Provision the New AWS Machine'

Comment (mkangia) [reply environments/staging/terraform.yml]: Thank you @pr33thi

**Pull request merged**